api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -153,6 +153,19 @@ message Transformation {
 // Extractions can be used to extract information from the request/response.
 // The extracted information can then be referenced in template fields.
 message Extraction {
+  // Define the Mode enum within the Extraction message
+  enum Mode {
+    // Default mode. Extract the value of the subgroup-th capturing group.
+    EXTRACT = 0; 
+    // Replace the value of the subgroup-th capturing group with the replacement_text.
+    // Note: replacement_text must be set for this mode.
+    SINGLE_REPLACE = 1;
+    // Replace all matches of the regex in the source with the replacement_text.
+    // Note: replacement_text must be set for this mode.
+    // Note: subgroup is ignored for this mode. configuration will fail if subgroup is set.
+    // Note: restrictions on the regex are different for this mode. See
```

Comment:
Complete this comment

Reply:
Updated here: https://github.com/solo-io/envoy-gloo/commit/f747a124c15c56a09d6436607f4efc992e37859d

```diff
-    // Note: restrictions on the regex are different for this mode. See
+    // Note: restrictions on the regex are different for this mode. See the regex field for more details.
```